docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,252 +8,324 @@ weight: 200
+   For example:
 
-Example of how to build and run:
+   ```bash
+    version: "3.8"
+    services:
+      grafana:
+      image: grafana/grafana-enterprise
+      container_name: grafana
+      restart: unless-stopped
+      ports:
+        - '3000:3000'
+   ```
 
-```bash
-cd packaging/docker/custom
-docker build \
-  --build-arg "GRAFANA_VERSION=latest" \
-  --build-arg "GF_INSTALL_IMAGE_RENDERER_PLUGIN=true" \
-  -t grafana-custom .
+1. To run `docker-compose.yaml`, run the following command:
 
-docker run -d -p 3000:3000 --name=grafana grafana-custom
-```
+   ```bash
+   # first go into the directory where you have created this docker-compose.yaml file
+   cd /path/to/docker-compose-folder 
 
-## Migrate from previous Docker containers versions
+   # then start the grafana container 
+   docker compose up -d docker-compose.yaml
+   ```
 
-This section contains important information if you want to migrate from previous Grafana container versions to a more current one.
+   Where:
 
-### Migrate to v7.3 or later
+   d = detached mode
+   
+   up = to bring the container up and running
 
-The Grafana Docker image runs with the `root` group (id 0) instead of the `grafana` group (id 472), for better compatibility with OpenShift. If you extend the official Docker image you may need to change your scripts to use the `root` group instead of `grafana`.
+To determine that Grafana is running, open a browser window and type `IP_ADDRESS:3000`. The sign in screen should appear.
 
-### Migrate to v6.5 or later
+### Stop the Grafana container
 
-Grafana Docker image now comes in two variants, one [Alpine](http://alpinelinux.org) based and one [Ubuntu](https://ubuntu.com/) based, see [Image Variants](#image-variants) for details.
+To stop the Grafana container, run the following command:
 
-### Migrate to v6.4 or later
+```bash
+docker compose down
+```
 
-Grafana Docker image was changed to be based on [Alpine](http://alpinelinux.org) instead of [Ubuntu](https://ubuntu.com/).
+> **Note:** For more information about using Docker Compose commands, refer to [docker compose](https://docs.docker.com/engine/reference/commandline/compose/).
 
-### Migrate to v5.1 or later
+### Save your Grafana data
 
-The Docker container for Grafana has seen a major rewrite for 5.1.
+When you use Docker containers, their data is temporary by default. If you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
 
-**Important changes**
+#### Use persistent storage (recommended)
 
```

Comment:
```suggestion

It is recommended to have persistent storage because without it, all data will be lost once the container is shut down. Use this method when you deploy in a production environment or when the data is critical and want the Docker service to fully manage the storage volume.

```